build-support/generate-bitcodes.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# generate-bitcodes.sh
+#
+# Generate bytecode handlers bitcode file, if necessary.
+# If the file is already present in the provided directory,
+# this script is a no-op.
+
+# TODO(Kyle): above we assume bash is an available shell, should we be doing this?
+
+# The root path of the project (i.e. noisepage/)
+PROJECT_ROOT=$1
+# The CMake artifact directory (i.e. noisepage/build)
+BINARY_DIR=$2
+# The path to clang
+CLANG_PATH=$3
+
+# Determine if the file is already present
+if [ -f "$BINARY_DIR/bin/bytecode_handlers_ir.bc" ]; then
+  exit 0
+fi
+
+# Run the compiler
+${PROJECT_ROOT}/build-support/tpl_bytecode_handlers_ir_compiler.py ${CLANG_PATH} ${BINARY_DIR} ${PROJECT_ROOT}/util/execution/bytecode_handlers_ir.cpp ${BINARY_DIR}/bin/bytecode_handlers_ir.bc || exit 1
```

Comment:
I'm generally a fan of printing out the bash commands that get executed by scripts. `set +x` or similar?

Reply:
Good point. I think I am going to go with manually `echo`ing though because it just looks a little nicer.